bridge/workflows/send_payouts.py

```diff
@@ -66,13 +68,25 @@ def mark_permanent_failure(request, error_message):
 	for request in requests_to_send:
 		conversion_rate_calculator = conversion_rate_calculator_factory.try_create_calculator(request.transaction_height)
 		if not conversion_rate_calculator:
-			mark_skipped(request, 'missing data')
+			mark_skipped(request, f'missing data at height {request.transaction_height}')
 			continue
 
 		logger.info('processing payout: %s:%s', request.transaction_hash, request.transaction_subindex)
-		conversion_function = conversion_rate_calculator.to_conversion_function(is_unwrap_mode)
+		conversion_function = conversion_rate_calculator.to_conversion_function(execution_context.is_unwrap_mode)
+		send_result = None
 		try:
-			send_result = await _send_payout(sender, network, request, conversion_function, fee_multiplier)
+			prepare_send_result = prepare_send(network, request, conversion_function, fee_multiplier)
+			if prepare_send_result.error_message:
+				send_result = TrySendResult(True, None, None, None, prepare_send_result.error_message)
+			else:
+				(is_exceeded, amount_remaining) = is_daily_limit_exceeded(network, database, prepare_send_result.transfer_amount)
+				if is_exceeded:
+					mark_skipped(request, f'daily limit exceeded ({amount_remaining} > {prepare_send_result.transfer_amount})')
```

Comment:
It shouldn't be instead?
`daily limit exceeded ({amount_remaining} < {prepare_send_result.transfer_amount})`
In that case amount_remaining is less than transfer_amount